src/services/imports.ts

```diff
@@ -1,5 +1,43 @@
 import { SourceCode } from "eslint";
-import { ImportDeclaration } from "estree";
+import {
+  Node,
+  Program,
+  ImportSpecifier,
+  ImportDeclaration,
+  ImportDefaultSpecifier,
+  ImportNamespaceSpecifier,
+} from "estree";
+
+const DEFAULT_SORT_INDEX = 100;
+
+type ImportSpecifierNode =
+  | ImportSpecifier
+  | ImportDefaultSpecifier
+  | ImportNamespaceSpecifier;
+
+export type CalculateSortOpts = {
+  sortBySpecifier?: boolean;
+  disableLineSorts: boolean;
+};
+
+export function getImportsWithNodesBetween(program: Program) {
```

Comment:
Not sure what the purpose of this logic? its trying to handle 

```
import A from 'b'
const a = 5
import B from 'b'
```

but is it really required?

Reply:
Yes. I faced with the issue after the first usage of the plugin with the next code:
```js
import pq from 'proxyquire';
const proxyquire = pq.noCallTq();
import { a, b } from 'somewhere';
// imports...
```
After the first plugin usage with this code, all of the variables between imports (like `proxyquire`) were deleted. Seems like it's unexpected behavior, because inside `createGetSortIndex` function the situation is handled with the default sort-index.

Reply:
Ohh 😮 that is bad, thanks for fixing that!